mock/py/mockbuild/config.py

```diff
@@ -728,6 +730,56 @@ def load_defaults(uidManager, version, pkg_python_dir):
     return setup_default_config_opts(gid, version, pkg_python_dir)
 
 
+def print_description(config_path, config_filename, uidManager, version, pkg_python_dir):
+    (basename, dirname, basename_without_ext) = parse_config_filename(config_filename)
+    try:
+        config_opts = load_config(config_path, config_filename, uidManager, version, pkg_python_dir)
+        description = config_opts.get("description", "")
+    except exception.ConfigError:
+        config_opts = load_defaults(uidManager, version, pkg_python_dir)
+        description = 'error during parsing the config file'
+    print("{} {}".format(basename_without_ext.ljust(34), description))
+
+
+def parse_config_filename(config_filename):
+    """ Accepts filename and returns (dirname, basename, basename_without_ext) """
+    basename = os.path.basename(config_filename)
+    dirname = os.path.dirname(config_filename)
+    basename_without_ext = os.path.splitext(basename)[0]
+    return (basename, dirname, basename_without_ext)
+
+def get_global_configs(config_opts):
+    """ Return filenames of global configs of chroots"""
+    result = []
+    for config_filename in glob("{}/*.cfg".format(config_opts['config_path'])):
+        if config_filename in ["/etc/mock/chroot-aliases.cfg", "/etc/mock/site-defaults.cfg"]:
+            continue
+        result.append(config_filename)
+    return result
+
+
+def get_user_configs(config_opts, uidManager):
+    """ Return filenames of user configs of chroots """
+    custom_path = os.path.join(os.path.expanduser('~' + pwd.getpwuid(uidManager.unprivUid)[0]), '.config/mock/*.cfg')
+    result = glob(custom_path)
+    return result
+
+
+@traceLog()
+def list_configs(config_opts, uidManager, version, pkg_python_dir):
+    log = logging.getLogger()
+    log.disabled = True
+    # array to save config paths
+    print("Global configs:")
```

Comment:
I would print the Custom configs first, just because there are a few of them ... and are kind of more "interesting" usually (if something goes wrong with mock config, or so).

Reply:
The list is long. So I welcome the interesting things towards the end - no need to scroll.

Reply:
I would hate waiting till the end of the command, honestly.  But if the performance will be fixed.